eva/executor/create_executor.py

```diff
@@ -34,11 +34,20 @@ def exec(self):
         Calls the catalog to create metadata corresponding to the table.
         Calls the storage to create a spark dataframe from the metadata object.
         """
+
+        print(f"CreateExecutor: inside exec")
+        print(f"CreateExecutor: dir(node) = {dir(self.node)}")
+        print(f"CreateExecutor: node.if_not_exists = {self.node.if_not_exists}")
+
+        # TODO: Disabling the below code for now. Check what actually needs to be done here.
+        '''
```

Comment:
I had to disable this code segment to get the "CREATE TABLE" query working. Need to check what exactly has to be done here

Reply:
Fixed